core/CMakeLists.txt

```diff
@@ -50,23 +40,18 @@ endif()
 
 # Include sources from subdirectories
 include(utils/CMakeLists.txt)
+
+if (DEFINED MODAL_REACTORS)
 include(modal_models/CMakeLists.txt)
+endif()
 
 # Print sources used for compilation
 list(JOIN REACTORC_SOURCES ", " PRINTABLE_SOURCE_LIST)
 message(STATUS "Including the following sources: " ${PRINTABLE_SOURCE_LIST})
 
-# Create the reactor-c library. If we are targeting Zephyr we have to use the
-# Zephyr Cmake extension to create the library and add the sources.
-if(PLATFORM_ZEPHYR)
-    message("--- Building Zephyr library")
-    zephyr_library_named(reactor-c)
-    zephyr_library_sources(${REACTORC_SOURCES})
-    zephyr_library_link_libraries(kernel)
-else()
-    add_library(reactor-c)
-    target_sources(reactor-c PRIVATE ${REACTORC_SOURCES})
-endif()
```

Comment:
Just curious: where did the Zephyr-specific stuff go? To be clear, I'm happy to see it gone.

Reply:
Thanks to your nice refactoring of all platform specific things into a separate target (low-level-platform) I can now do the zephyr-specific things there :D